### PR TITLE
sl-239: Fix search in sites page

### PR DESCRIPTION
**Pull request by vbazhynov**
image
image

### CODE REVIEW

**Review by Alyona-Pisotska — APPROVED** [review]

File: frontend/src/pages/sites/sites.tsx [hunk 1]
```diff
@@ -80,21 +89,25 @@ const Sites: React.FC = () => {
     },
   );
 
+  const showSites = useMemo(() => {
```
Comment (what1s1ove) [inline frontend/src/pages/sites/sites.tsx]:
```suggestion
  const hasSites = useMemo(() => {
```

File: frontend/src/pages/sites/sites.tsx [hunk 2]
```diff
@@ -33,6 +35,8 @@ const Sites: React.FC = () => {
   const dispatch = useAppDispatch();
   useTitle('My sites');
 
+  const [isSearching, setIsSearching] = useState(false);
```
Comment (what1s1ove) [inline frontend/src/pages/sites/sites.tsx]: type a state

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/pages/sites/sites.tsx (hunk 1 above)
Comment (vbazhynov) [reply frontend/src/pages/sites/sites.tsx]: We already have the hasSites variable, it shows the result of checking for the presence of sites in the array of sites after the fetch.

Comment (what1s1ove) [reply frontend/src/pages/sites/sites.tsx]: okay, should be a different name. `show` is a noun. boolean variables should start with `yes`. Ex `has`, `is`, etc

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/pages/sites/sites.tsx (hunk 1 above)
Comment (vbazhynov) [reply frontend/src/pages/sites/sites.tsx]: ready

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**